examples/react-client/fishjam-chat/src/components/DevicePicker.tsx

```diff
@@ -52,6 +52,22 @@ const DeviceSelect: FC<DeviceSelectProps> = ({ device }) => {
           Stream
         </Button>
       )}
+      <Button
+        disabled={!!device.stream}
+        onClick={async () => {
+          await device.initialize();
+        }}
+      >
+        Start
+      </Button>
+      <Button
+        disabled={!device.stream}
+        onClick={async () => {
+          await device.stop();
+        }}
+      >
+        Stop
```

Comment:
I know it is example app - but can we make these function more descriptive? Right now UI could have two stop buttons at some point.

Reply:
I've changed it to `Stop / Start streaming` and `Stop / Start device`